tools/scripts/lib_install/check-cmake-version.sh

```
@@ -0,0 +1,17 @@
+!/usr/bin/env bash
```

Shellcheck (0.10.0)

[error] Use #!, not just !, for the shebang.

(SC1104)